Gemfile

```diff
@@ -52,6 +52,7 @@ gem "text_helpers"
 gem "chosen-rails"
 gem "fine_uploader", path: "vendor/engines/fine_uploader"
 gem "rubyzip"
+gem "momentjs-rails"
```

Comment:
What are we using this gem for?

Reply:
It's a dependency of the new version of FullCalendar. Probably worth a comment around it. I was hoping to use a gem version of full calendar, but the existing one out there is out of date (https://github.com/bokmann/fullcalendar-rails). I might switch to using the rails-assets version, though, though I doubt that has the moment dependency.

Reply:
There is a PR in that gem for the upgrade to 3.9, but it hasn't been merged, but we could track to it.